Error while cargo test except for the root directory.

`cargo test -p codechain-core` in root directory works normally, but `cargo test` in the core and some directory occurs the following error:
```
   Compiling codechain-key v0.1.0 (file:///Users/redongjun/workspace/codechain/key)
   Compiling codechain-vm v0.1.0 (file:///Users/redongjun/workspace/codechain/vm)
   Compiling codechain-keystore v0.1.0 (file:///Users/redongjun/workspace/codechain/keystore)
   Compiling codechain-network v0.1.0 (file:///Users/redongjun/workspace/codechain/network)
error[E0432]: unresolved import `ckey::verify`
  --> vm/src/executor.rs:18:12
   |
18 | use ckey::{verify, Signature, SignatureData, SIGNATURE_LENGTH};
   |            ^^^^^^ no `verify` in the root

...

warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/core/Cargo.toml
+++ b/core/Cargo.toml
@@ -8,7 +8,7 @@ byteorder = "1.2.3"
 codechain-crypto = { path = "../crypto" }
 codechain-io = { path = "../util/io" }
 codechain-json = { path = "../json" }
-codechain-key = { path="../key" }
+codechain-key = { path = "../key", features = ["ecdsa"] }
 codechain-keystore = { path="../keystore" }
 codechain-logger = { path = "../util/logger" }
 codechain-merkle = { path = "../util/merkle" }

--- a/discovery/Cargo.toml
+++ b/discovery/Cargo.toml
@@ -5,7 +5,7 @@ authors = ["CodeChain Team <codechain@kodebox.io>"]
 
 [dependencies]
 codechain-crypto = { path = "../crypto" }
-codechain-key = { path="../key" }
+codechain-key = { path = "../key", features = ["ecdsa"] }
 codechain-logger = { path = "../util/logger" }
 codechain-network = { path = "../network" }
 codechain-types = { path = "../primitives/codechain-types" }

--- a/keystore/Cargo.toml
+++ b/keystore/Cargo.toml
@@ -7,7 +7,7 @@ authors = ["CodeChain Team <codechain@kodebox.io>", "Parity Technologies <admin@
 log = "0.4.1"
 libc = "0.2"
 rand = "0.5.3"
-codechain-key = { path = "../key" }
+codechain-key = { path = "../key", features = ["ecdsa"] }
 codechain-types = { path = "../primitives/codechain-types" }
 serde = "1.0"
 serde_json = "1.0"

--- a/network/Cargo.toml
+++ b/network/Cargo.toml
@@ -7,7 +7,7 @@ authors = ["CodeChain Team <codechain@kodebox.io>"]
 codechain-crypto = { path = "../crypto" }
 codechain-finally = { path = "../util/finally" }
 codechain-io = { path = "../util/io" }
-codechain-key = { path="../key" }
+codechain-key = { path = "../key", features = ["ecdsa"] }
 codechain-logger = { path = "../util/logger" }
 codechain-types = { path = "../primitives/codechain-types" }
 log = "0.4.1"

--- a/rpc/Cargo.toml
+++ b/rpc/Cargo.toml
@@ -7,7 +7,7 @@ authors = ["CodeChain Team <codechain@kodebox.io>"]
 
 [dependencies]
 codechain-core = { path = "../core" }
-codechain-key = { path = "../key" }
+codechain-key = { path = "../key", features = ["ecdsa"] }
 codechain-logger = { path = "../util/logger" }
 codechain-network = { path = "../network" }
 codechain-types = { path = "../primitives/codechain-types" }
@@ -25,4 +25,3 @@ jsonrpc-core = { git = "https://github.com/paritytech/jsonrpc.git", branch = "pa
 jsonrpc-macros = { git = "https://github.com/paritytech/jsonrpc.git", branch = "parity-1.11" }
 jsonrpc-http-server = { git = "https://github.com/paritytech/jsonrpc.git", branch = "parity-1.11" }
 jsonrpc-ipc-server = { git = "https://github.com/paritytech/jsonrpc.git", branch = "parity-1.11" }
-

--- a/sync/Cargo.toml
+++ b/sync/Cargo.toml
@@ -7,7 +7,7 @@ authors = ["CodeChain Team <codechain@kodebox.io>"]
 
 [dependencies]
 codechain-core = { path = "../core" }
-codechain-key = { path = "../key" }
+codechain-key = { path = "../key", features = ["ecdsa"] }
 codechain-logger = { path = "../util/logger" }
 codechain-merkle = { path = "../util/merkle" }
 codechain-network = { path = "../network" }
@@ -22,4 +22,4 @@ time = "0.1"
 triehash = { path = "../util/triehash" }
 
 [dev-dependencies]
-codechain-key = { path = "../key" }
+codechain-key = { path = "../key", features = ["ecdsa"] }

--- a/vm/Cargo.toml
+++ b/vm/Cargo.toml
@@ -7,7 +7,7 @@ authors = ["CodeChain Team <codechain@kodebox.io>"]
 
 [dependencies]
 codechain-crypto = { path = "../crypto" }
-codechain-key = { path = "../key" }
+codechain-key = { path = "../key", features = ["ecdsa"] }
 codechain-types = { path = "../primitives/codechain-types" }
 
 [dev-dependencies]

